app/views/shared/forms/received_benefit_confirmation/_form.html.erb

```diff
@@ -1,12 +1,14 @@
 <%= form_with(model:, url:, method: :patch, local: true) do |form| %>
 
-  <%= form.govuk_collection_radio_buttons(:passporting_benefit, Providers::ReceivedBenefitConfirmationForm.radio_options, :value, :label,
-                                          legend: { size: "xl", tag: "h1", text: content_for(:page_title) }) %>
+  <%= form.govuk_collection_radio_buttons(:passporting_benefit, Providers::ReceivedBenefitConfirmationForm.radio_options,
+                                          :value, :label,
+                                          legend: { size: "xl", tag: "h1", text: content_for(:page_title) },
+                                          classes: ["govuk-!-margin-top-4 govuk-!-margin-bottom-0"]) %>
 
-    <%= form.govuk_radio_divider %>
-    <%= form.govuk_radio_button(:passporting_benefit, :none_selected, label: { text: t("generic.none_selected") }) %>
+  <%= form.govuk_radio_divider %>
+  <%= form.govuk_radio_button(:passporting_benefit, :none_selected, label: { text: t("generic.none_selected") }) %>
 
-  <div class="govuk-!-padding-bottom-3"></div>
+  <br>
```

Comment:
nit: for layout purposes, it may be best to use [govuk spacing](https://design-system.service.gov.uk/styles/spacing/#static-spacing) overrides on either  sibling elements